ngraph/core/include/ngraph/op/proposal.hpp

```diff
@@ -79,10 +79,37 @@ namespace ngraph
                 const ProposalAttrs& get_attrs() const { return m_attrs; }
                 virtual bool visit_attributes(AttributeVisitor& visitor) override;
 
-            private:
+            protected:
                 ProposalAttrs m_attrs;
             };
         }
+
+        namespace v4
+        {
+            class NGRAPH_API Proposal : public op::v0::Proposal
+            {
+            public:
+                static constexpr NodeTypeInfo type_info{"Proposal", 4};
```

Comment:
Use new RTTI declarations

Reply:
What are the new RTTI declaraions? Can you give reference example?

Reply:
Example: https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/include/ngraph/op/relu.hpp#L37

Reply:
Done.